src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14Records.java

```diff
@@ -24,15 +35,15 @@ public record AnnotatedBinaryNode(@NonNull1 Node left, @NonNull1 Node right) { }
 
     //Implements interface
     public record Polar(double r, double theta) implements Coords {
-        @Override
-        public double x() {
-            return r * Math.cos(theta);
-        }
+    @Override
```

Comment:
looks like this record's indentation got off (I assume unintentionally?)

Reply:
Fixed